price-reporter/src/utils.rs

```diff
@@ -128,6 +131,8 @@ pub struct PriceReporterConfig {
     pub remap_chain: Chain,
     /// The configuration options that may be used by exchange connections
     pub exchange_conn_config: ExchangeConnectionsConfig,
+    /// The HMAC key for the admin API
+    pub admin_key: HmacKey,
```

Comment:
can we make this optional and disable the refresh endpoint if that's the case? helpful for testing locally